src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -814,7 +821,18 @@ def forward(
             encoder_hidden_shape = (encoder_batch_size, encoder_sequence_length)
             if encoder_attention_mask is None:
                 encoder_attention_mask = torch.ones(encoder_hidden_shape, device=device)
-            encoder_extended_attention_mask = self.invert_attention_mask(encoder_attention_mask)
```

Comment:
Not sure why in the first place this is done at the layer level and not model level. This is an identical call at all layers.

Reply:
I believe this is done at the model level, inside `BertModel#forward()`